model/test_results.go

```diff
@@ -81,6 +81,11 @@ type TestResults struct {
 	// This is an optimization for Evergreen's UI features that display a
 	// limited number of failing tests for a task.
 	FailedTestsSample []string `bson:"failed_tests_sample"`
+	// Migration holds statistics for the BSON to Parquet backfill, it is
+	// only present in the temporary test results migration collection.
+	//
+	// TODO (EVG-16140): Remove once we do the  BSON to Parquet cutover.
+	Migration *MigrationStats `bson:"migration,omitempty"`
```

Comment:
Note to self: do we need this or would a separate collection be easier?

Reply:
I was thinking the same thing, but adding this as a sub-doc to the test results docs in the temp collection is programmatically simpler and I don't see any major benefit of putting it in a separate collection.